js/src/forum/pages/labels/RecipientLabel.tsx

```diff
@@ -18,7 +18,10 @@ export default class RecipientLabel extends Component<IRecipientLabelAttrs> {
 
         newAttrs.style = newAttrs.style || {};
         newAttrs.className = classList('RecipientLabel', newAttrs?.className);
-        newAttrs.href = link;
+
+        if (link) {
+            newAttrs.href = app.route.user(recipient);
```

Comment:
Although this will certainly fix the `true` link if the route is to a `user`, but what about when the recipient is a `group` instead?

Reply:
My last commit should fix that now.